src/components/Articles/Articles.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from "react";
+import { getArticles } from "../../api";
+
+const Articles = () => {
+  const [total, setTotal] = useState(0);
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getArticles()
+      .then(({ articles, total_count }) => {
+        setArticles(articles);
+        setTotal(total_count);
```

Comment:
I'm not 100% sure what this is doing? I see that we're setting the total, but then don't seem to be using the state of total anywhere?